### DISCUSSION TITLE
About attribution, can I remove it or not in commercial projects?

### DISCUSSION THREAD

**Discussion by luciancaetano** [xyflow/xyflow#2520] [Q&A] [answered]
About attribution, can I remove it or not in commercial projects?

I'm having a certain difficulty because it seems there is a license beyond MIT.
After all, in commercial projects, should I pay a subscription to be able to remove the attribution? or you could simply remove it, and the signature would be a way for the company(user of library) to thank s the maintener of the project by helping to maintain it ?

**Comment by moklick** [comment, marked as answer]
It's a regular MIT license. You don't need to pay to remove the attribution.
